app/constants.ts

```diff
@@ -1,5 +1,6 @@
 export const GEO_API_KEY = '07544e7bd3442c641181e2b651145b42';
 export const GEO_URL = 'http://api.openweathermap.org/data/2.5/';
 
-export const geoLocation = document.getElementById('geolocation');
-export const googleMap = document.getElementById("google-map");
\ No newline at end of file
+export const NUMBER_OF_CITIES = 20;
+
+export const googleMapElement = document.getElementById("google-map");
```

Comment:
@LeraKruchek I tried to export selector, but it didn't work for google map, then just forget to remove from constants file.